### DISCUSSION TITLE
Does this project really use RocksDB in android ?

### DISCUSSION THREAD

**Discussion by Hesowcharov** [react-native-async-storage/async-storage#567] [General]
## You want to:  
Can't find any approves in the code that the library will use RocksDB in android. Is this relevant information from the documentation?

## Details:
https://github.com/react-native-async-storage/async-storage/blob/af0677c43e5087404b0def0dd084a69e1930cbf5/types/index.d.ts#L12-L15

**Comment by krizzu** [comment]
Hey,

SQLite is used on Android. This comment must be a leftover from old times.
